### ISSUE TITLE
abuout xy and wh in gen_sineembed_for_position

### ISSUE THREAD

**Issue by xianjiez** [IDEA-opensource/DAB-DETR#79]
I have a question about the correspondence between xy and wh in torch.cat((pos_y, pos_x, pos_w, pos_h), dim=2), why isn't it torch.cat (pos_y, pos_x, pos_h, pos_w)， This way, y corresponds to h, and x corresponds to w, I hope to receive your help.
